cos/templates/game.jinja2

```diff
@@ -1,33 +1,75 @@
 <!DOCTYPE html>
 <html lang="{{request.locale_name}}">
   <head>
+{#      <script type="text/javascript" src="{{request.static_url('cos:static/jquery-3.2.0.min.js')}}"></script>#}
```

Comment:
This line can be removed as it is just commented out code that is no longer used.

Reply:
fixed